frameworks/g-cloud-9/questions/services/lotName.yml

```diff
@@ -0,0 +1,2 @@
+question: 'Product type'
+type: text
```

Comment:
In short... I'm not sure exactly what would break if we don't have it, but I'm pretty sure *something* will.